index.html

```diff
@@ -4,16 +4,91 @@
 <head>
   <meta charset="utf-8">
 
-  <title>My Resume</title>
+  <title>LV Wood Resume</title>
 
   <link rel="stylesheet" href="css/index.css">
+  <link href="https://fonts.googleapis.com/css?family=Montserrat|Playfair+Display" rel="stylesheet">
 
   <!--[if lt IE 9]>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
 </head>
 
 <body>
-  <h1>My Resume</h1>
+<div class="container">
+    
+<div class="header-nav">
+    <header>
+        <h1>Leslee V. Wood</h1>
+        <nav>
+          <a href="">Experience</a>
+          <a href="">Portfolio</a>
+          <a href="">Education</a>
+          <a href="">Contact</a>
+        </nav>
+    </header>
+</div>
+
+<div>
+  <div class="main-content">
+    <div class="content-section">
+    <section>
+      <h2>Introduction</h2>
+      <p>paragraph bio</p>
+      <h3>Skills</h3>
+      <li>
+        <ul>HTML and CSS</ul>
+        <ul>Adobe Creative Cloud: Illustrator, InDesign, Photoshop, Premiere, Audition</ul>
+        <ul>Site-building tools: Wix, Weebly, Wordpress</ul>
+        <ul>Additional Creative Programs: Finale, Sibelius, Logic-Pro</ul>
+      </li>
+    </section>
+  </div>
+  <div class="content-section">
+    <section>
+      <h2>Experience</h2>
+      <h3>Lambda School Full-Stack Web Development Program</h3>
+      <p>brief explanation</p>
+      <h3>Graphic Designer - KU School of Music</h3>
+      <p>brief explanation</p>
+      <h3>Webmaster - KC VITAs</h3>
+      <p>brief explanation</p>
+      <h3>Freelance Graphic Designer</h3>
+      <p>brief explanation</p>
+    </section>
+  </div>
+  <div class="content-section">
+    <section>
+      <h2>Portfolio</h2>
+      <div class="project-boxes">
+        <img src="" alt="KC VITAs website">
+        <img src="" alt="Ignea Strata Website">
+        <img src="" alt="Art of the Troubadours">
+        <img src="" alt="Sweet Eats">
+        <img src="" alt="Great Idea">
+        <img src="" alt="Greek Lambda">
+      </div>
+      <div class="portfolio-links">
+        <button> <a href="https://lvwood.github.io/"></a>
+          view online portfolio</button>
+      </div>
+    </section>
```

Comment:
For button links, you might want to just stick to using an `<a>` tag (no need to put it inside a `<button>` tag) and then styling it to look similar to a button. Also, a heads-up for you to put your text inside the `<a></a>` tags as well - clicking the text won't bring you to the linked page if the text is not inside the tags.